src/entrypoint/graphql.ts

```diff
@@ -357,9 +357,12 @@ export async function startApolloServer() {
           { lastConnection: new Date() },
           { new: true },
         )
-        if (yamlConfig.proxyChecking.enabled) {
-          fetchIPDetails({ ip, user, logger })
+
+        if (yamlConfig.ipChecking.enabled) {
+          const checkProxy = yamlConfig.ipChecking.proxyChecking.enabled
+          fetchIPDetails({ ip, user, logger, checkProxy })
```

Comment:
I think we could just initiate `checkProxy` within `fetchIPDetails` directly if the only reason `checkProxy` const is being created here is to pass it to the `fetchIPDetails` function.